SonarScanner.Shim/SonarScanner.Wrapper.cs

```diff
@@ -17,13 +17,13 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
- 
-using SonarQube.Common;
```

Comment:
Sorry I have the automatic sorting on save... Can undo if needed.

Reply:
Let us all adopt the VS settings favored by the .NET team. But it should be documented somewhere.  Maybe here: https://github.com/SonarSource/sonar-developer-toolset

Reply:
We could also enforce it with a [StyleCopAnalyzer](https://github.com/DotNetAnalyzers/StyleCopAnalyzers]). I could open a PR for this if the team would be interested in this.